src/main/groovy/jenkins/automation/utils/BaseJobBuilder.groovy

```diff
@@ -33,7 +33,11 @@ class BaseJobBuilder {
                 numToKeep(10)
             }
             publishers{
-                mailer emails.join(' ')  //TODO use extended email
+                allowBrokenBuildClaiming()
+
+                if(emails){
```

Comment:
Groovy's truthiness will treat an empty array as null, thus this check is valid. Perfectly good reasons exist for not requiring email notifications on job failures